egs2/aishell4/asr1/local/data.sh

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/env bash
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+. ./db.sh || exit 1;
+
+ . utils/parse_options.sh || exit 1;
```

Comment:
You should move it after `stage=0` otherwise the argument cannot be parsed into correct variable